docs/source/yelpsoa_configs.rst

```diff
@@ -111,6 +111,8 @@ instance MAY have:
     Parsing the Marathon config file will fail if both args and cmd are
     specified [#note]_.
 
+  .. _env:
```

Comment:
That was to allow the link from the workflow doc directly to the env bullet. Otherwise, the closest we can link is #marathon-clustername-yaml. If you are fine with the less specific link, I'll make the change.